src/index.js

```diff
@@ -1,36 +1,113 @@
+/* eslint-disable import/no-cycle */
 import './style.css';
+import updating from './modules/updating.js';
 
 const todoList = document.querySelector('.todo-list');
+const todoTask = document.getElementById('todo-task');
+const addTaskBtn = document.getElementById('add-task-btn');
+let storedToDos = [];
+let editIndex = null;
 
-const taskStorage = [
-  {
-    id: 0,
-    description: 'Watch some courses',
-    completed: true,
-  },
-  {
-    id: 1,
-    description: 'Watch some tuto',
-    completed: true,
-  },
-  {
-    id: 2,
-    description: 'Complete the week',
-    completed: false,
-  },
-];
-
-const autoFillData = () => {
+const getToDos = () => {
+  if (localStorage.getItem('toDos') === null) {
+    storedToDos = [];
+  } else {
+    storedToDos = JSON.parse(localStorage.getItem('toDos'));
+  }
   let show = '';
-  taskStorage.forEach((i) => {
-    show += `<li class="task-item" id="${i.id}">
-        <div>
-            <input type="checkbox" class="list-input" id="${i.id} ${i.completed ? 'checked' : ''}">
-            <label type="text" class="list-label" for="${i.id}" id="${i.id}">${i.description}</label>
-        </div>
-        <i class="fas fa-ellipsis-v resize" id="${i.id}"></i>
-        </li>`;
+  storedToDos.forEach((task, index) => {
+    const completed = task.completed ? 'line-through' : '';
+    show += `
+    <li class="task-item ${completed}" id=${index}>
+    <div>
+    <input class="list-input input checkbox" type="checkbox" id=${index} ${task.completed ? 'checked' : ''}>
+    <label for="${index}" class='list-label label'>${task.description}</label>
+    </div>
+    <div>
+    <i class="fas fa-trash delete resize" id=${index}></i>
+    <i class="fas fa-edit edit resize" id=${index}></i>
+    </div> 
+    </li>`;
   });
   todoList.innerHTML = show;
 };
-autoFillData();
\ No newline at end of file
+
+const saveToDos = ({ index, description, completed = false }) => {
```

Comment:
- [x] _Index_ property of added item does not match new array length. For instance, if a 5th item is added to the list, the index of that item should equal to 5. Kindly ensure to improve upon the same.

**Mismatch in index property and array length:**
![image](https://user-images.githubusercontent.com/12745474/213357818-8bc2858e-d495-4bdd-85a5-14ad3f4768e7.png)

Reply:
well resolved! ✔️